### PR TITLE
Initial README commit

### PR DESCRIPTION
**Pull request by xiamaz**
Only product vision from the initial meeting. More to come, when technical specifics have been discussed in the group.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# FlyINN
+
+Intermodal mobility app for a sharing economy.
+
+## Product vision
+
+Seamless access to personal mobile device through an external screen integrated
+into an external screen, which allows comprehensive control of the device.
```
Comment (lbb) [inline README.md:8]: We need to mention the License

Comment (lbb) [reply README.md:8]: (Also separate the license section with a horizontal ruler)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# FlyINN
```
Comment (lbb) [inline README.md]:
```suggestion
# FlyInn
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+Intermodal mobility app for a sharing economy.
```
Comment (lbb) [inline README.md]: Better describe what FlyInn is. IMHO this does not bring across the purpose or functionality.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+## Product vision
```
Comment (lbb) [inline README.md]: Maybe merge this with the introduction to FlyInn

Comment (xiamaz) [reply README.md]: Handled in commit f1f8a54

File: README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# FlyInn
+
+## Product vision
```
Comment (lbb) [inline README.md]:
```suggestion
```

**Review by lbb — CHANGES_REQUESTED** [review]

**Review by lbb — APPROVED** [review]
LGTM

**Pull request merged**